### ISSUE TITLE
[BUG]Networking is very unstable in CRC on Windows Server

### ISSUE THREAD

**Issue by jeffsaremi** [code-ready/crc#1129]
### General information

  * OS: Windows
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)? Yes

This is of course on Windows Server 2016 which is unsupported but I wanted to report sporadic behavior where the outward network connection is lost entirely while CRC vm is still running.

A restart of CRC does not recover at all. crc.exe is unable to connect period. Cannot do status/stop/ip or anything like that
```
PS C:\Users\jesaremi> .\crc-bin\crc status
ERRO Unable to connect to the server: dial tcp: lookup api.crc.testing: no such host
 - exit status 1
```
Checking the DNS addresses of Windows server:
```
PS C:\Windows\system32> Get-DnsClientServerAddress

InterfaceAlias               Interface Address ServerAddresses
                             Index     Family
--------------               --------- ------- ---------------
Embedded LOM 1 Port 2               13 IPv4    {}
Embedded LOM 1 Port 2               13 IPv6    {fec0:0:0:ffff::1, fec0:0:0:ffff::2, fec0:0:0:ffff::3}
Embedded LOM 1 Port 4                6 IPv4    {}
Embedded LOM 1 Port 4                6 IPv6    {fec0:0:0:ffff::1, fec0:0:0:ffff::2, fec0:0:0:ffff::3}
Embedded LOM 1 Port 3                5 IPv4    {}
Embedded LOM 1 Port 3                5 IPv6    {fec0:0:0:ffff::1, fec0:0:0:ffff::2, fec0:0:0:ffff::3}
vEthernet (HPE Ethernet 1...        14 IPv4    {}
vEthernet (HPE Ethernet 1...        14 IPv6    {fec0:0:0:ffff::1, fec0:0:0:ffff::2, fec0:0:0:ffff::3}
**vEthernet (crc)                     17 IPv4    {10.193.18.133}**
vEthernet (crc)                     17 IPv6    {}
Loopback Pseudo-Interface 1          1 IPv4    {}
Loopback Pseudo-Interface 1          1 IPv6    {fec0:0:0:ffff::1, fec0:0:0:ffff::2, fec0:0:0:ffff::3}
isatap.sys-sqlsvr.local              8 IPv4    {10.193.18.133}
isatap.sys-sqlsvr.local              8 IPv6    {}
Reusable ISATAP Interface...        12 IPv4    {}
Reusable ISATAP Interface...        12 IPv6    {fec0:0:0:ffff::1, fec0:0:0:ffff::2, fec0:0:0:ffff::3}
```
Can login using SSH and the ip address however. Login using ssh:
```
PS C:\Users\jesaremi> ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=NUL -i C:\Users\jesaremi\.crc\machines\crc\id_rsa core@10.193.18.133
Warning: Permanently added '10.193.18.133' (ECDSA) to the list of known hosts.
Red Hat Enterprise Linux CoreOS 43.81.202001142154.0
  Part of OpenShift 4.3, RHCOS is a Kubernetes native operating system
  managed by the Machine Config Operator (`clusteroperator/machine-config`).

WARNING: Direct SSH access to machines is not recommended; instead,
make configuration changes via `machineconfig` objects:
  https://docs.openshift.com/container-platform/4.3/architecture/architecture-rhcos.html
```

Inside the CRC VM, the resolution of 'api.crc.testing' fails:
```
[core@crc-w6th5-master-0 ~]$
[core@crc-w6th5-master-0 ~]$ nslookup api.crc.testing
Server:         10.193.8.10
Address:        10.193.8.10#53

** server can't find api.crc.testing: NXDOMAIN
```
The only way to recover the network is to do a reset as was suggested here earlier, or to delete and re-install crc completely
```Set-DnsClientServerAddress -InterfaceAlias "vEthernet (crc)" -ResetServerAddresses```

```
[core@crc-w6th5-master-0 ~]$ cat /etc/resolv.conf
# Generated by NetworkManager
search sys-sqlsvr.local
nameserver 10.193.8.10
nameserver 10.193.8.38
nameserver 10.158.162.254
# NOTE: the libc resolver may not support more than 3 nameservers.
# The nameservers listed below may not be recognized.
nameserver 10.158.162.253
```
Let me know if there's anything else I can report on this for you. thanks

**Comment by gbraad** [comment]
> which is unsupported 

but this looks like you use the override network, and therefore you might also have to force the local lookup. I do not see the dnsmasq entry... try to use `crc start -n 127.0.0.1` ?

**Comment by gbraad** [comment]
Have not been able to reproduce this. This might be caused by the configuration. Can you detail more about the setup of the `crc` virtual switch?

**Comment by jeffsaremi** [comment]
We tried the other solution too
We eventually decided that CRC is just not for us. 
We're going to pursue other ways of getting OpenShift for ourselves.
Closing this

**Comment by gbraad** [comment]
@jeffsaremi I am curious what the situation actually is, as from the issues you have tried many different things and some outside of the scope of what CRC serves. Would it be possible to set up a call to get some feedback? Perhaps, and this is what I mentioned earlier, using the installer to target Azure might have been a better fit. But from the issues I see there are restrictions in the hosts you can run this on...

**Comment by munger1985** [comment]
i have the same experience in linux, unstable, very much, u can email me, i can show u the unstability. jingsong.liu@oracle.com

**Comment by cfergeau** [comment]
> i have the same experience in linux, unstable, very much, u can email me, i can show u the unstability. [jingsong.liu@oracle.com](mailto:jingsong.liu@oracle.com)

Could you open a new issue for that with details?
